client/src/templates/Challenges/classic/Editor.js

```diff
@@ -96,6 +96,7 @@ class Editor extends Component {
   editorDidMount = (editor, monaco) => {
     this._editor = editor;
     this._editor.focus();
+		this._editor.getAccessibility();
```

Comment:
I cannot see this method in the [API docs](https://microsoft.github.io/monaco-editor/api/). Have you tested this locally?

I would prefer a keyboard shortcut to toggle the editor in to screen reader mode, which would also set a flag somewhere so that if the Editor Component unmounts and then remounts, the screen reader mode is enable again automatically

Reply:
Hey, so I looked into this and my fix actually came from code I changed in the monaco editor code, which is in the .gitignore since FCC just imports it as a node module. I'll look into submitting something to the Monaco people and get back with whatever insight I hear

Reply:
Thanks for the update. We are going to close this on that note, and lets open a fresh PR whenever you are ready.